lib/galaxy/datatypes/binary.py

```diff
@@ -908,3 +908,83 @@ def sniff( self, filename ):
             return False
 
 Binary.register_sniffable_binary_format('RData', 'RData', RData)
+
+
+class OxliBinary(Binary):
+
+    def __init__(self, **kwd):
+        Binary.__init__(self, **kwd)
+
+    def sniff(self, filename, filetype):
+        try:
+            with open(filename) as fileobj:
+                header = fileobj.read(4)
+                if binascii.b2a_hex(header) == binascii.hexlify('OXLI'):
+                    fileobj.seek(1)
+                    ftype = fileobj.read(1)
+                    if binascii.b2a_hex(ftype) == filetype:
+                        return True
+            return False
+        except IOError:
+            return False
+
+
+class OxliCountGraph(OxliBinary):
+
+    def __init__(self, **kwd):
+        OxliBinary.__init__(self, **kwd)
+
+    def sniff(self, filename):
+        return OxliBinary.sniff(self, filename, "01")
+
+
+class OxliNodeGraph(OxliBinary):
+
+    def __init__(self, **kwd):
+        OxliBinary.__init__(self, **kwd)
+
+    def sniff(self, filename):
+        return OxliBinary.sniff(self, filename, "02")
```

Comment:
I'm not sure, but maybe we should use super() here?

Depending on the filesize of an example dataset a doctest would be nice:

        """
        OxliNodeGraph starts with magic string "OXLI01"
        >>> from galaxy.datatypes.sniff import get_test_fname
        >>> fname = get_test_fname( 'sequence.csfasta' )
        >>> OxliNodeGraph().sniff( fname )
        False
        >>> fname = get_test_fname( 'khmer.oxli01' )
        >>> OxliNodeGraph().sniff( fname )
        True
        """